src/gflownet/envs/grid.py

```diff
@@ -64,24 +64,6 @@ def __init__(
         self.min_step_len = min_step_len
         self.max_step_len = max_step_len
         self.cells = np.linspace(cell_min, cell_max, length)
-        self.oracle = {
-            "default": None,
-            "cos_N": self.func_cos_N,
-            "corners": self.func_corners,
-            "corners_floor_A": self.func_corners_floor_A,
-            "corners_floor_B": self.func_corners_floor_B,
-        }[oracle_func]
-        if proxy:
-            self.proxy = proxy
-        else:
-            self.proxy = self.oracle
-        self.reward = (
-            lambda x: [0]
-            if not self.done
-            else self.proxy2reward(self.proxy(self.state2oracle(x)))
-        )
-        self._true_density = None
```

Comment:
Not really imp, but is there any reason we removed the initialization `self._true_density` from the `init()`?